multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -70,6 +70,9 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # serviceCIDR: "10.3.0.0/24"
 
 # CIDR for all pod IP addresses
+# this subnet must be at least as large as serviceCIDR * workerCount
```

Comment:
The serviceCIDR shouldn't have anything to do with the size of the podCIDR (they just cannot overlap).